.github/workflows/update_galata_references.yaml

```diff
@@ -0,0 +1,73 @@
+name: Update Galata References
+
+on:
+  issue_comment:
+    types: [created, edited]
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  update-reference-screenshots:
+    name: Update Galata References
+    if: ${{ github.event.issue.pull_request && contains(github.event.comment.body, 'update galata references') }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Checkout the branch from the PR that triggered the job
+        run: hub pr checkout ${{ github.event.issue.number }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
As noticed in https://github.com/jupyterlab/maintainer-tools/pull/39#issue-1111688219, it looks like this would make it possible for anyone to commit and push to the PR?

Reply:
Yes, but I am not sure the concerns about adding/executing arbitrary code apply here. The action really just commits the test references (it's hardcoded), so yeah anyone could comment "update galata references" and update the references, but that should not do much harm.

Reply:
And I think it's actually important for anyone to be able to do that. Because new contributors should be able to trigger that action in their own PR.

Reply:
Alright, this workflow is indeed more specific than the "PR Script" workflow provided by https://github.com/jupyterlab/maintainer-tools

Reply:
Your comment about adding such a tool in maintainer-tools makes sense, we should probably not copy paste this everywhere Galata is used. I'll see if I can make something.

Reply:
Done in https://github.com/jupyterlab/maintainer-tools/pull/41